cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -30,7 +30,8 @@
 
 @dataclass
 class RNADNACollection:
-    """Contains the id for an RNA sample, the name of its connected DNA sample, and a list of connected DNA cases."""
+    """Contains the id for an RNA sample, the name of its connected DNA sample,
+    and a list of connected, uploaded DNA cases."""
 
     rna_sample_id: str
     dna_sample_name: str
```

Comment:
```suggestion
    rna_sample_id: str
    dna_sample_name: str
```
rna_sample_id: str does this look for the RNA sample name, RNA sample entry id (database row) or the RNA sample internal id